kubectl-fdb/cmd/buggify_crash_loop.go

```diff
@@ -98,6 +112,72 @@ kubectl fdb -n default buggify crash-loop -c cluster pod-1 pod-2
 	return cmd
 }
 
+// updateCrashLoopContainerList updates the crash-loop container-list of the cluster
+func updateCrashLoopContainerList(kubeClient client.Client, clusterName string, containerName string, pods []string, namespace string, wait bool, clear bool, clean bool) error {
+	if len(containerName) == 0 {
+		return updateCrashLoopList(kubeClient, clusterName, pods, namespace, wait, clear, clean)
```

Comment:
I think we should drop the support for the old deprecated way and clean up our code.